scripts/CheckSecurityConfigs.s.sol

```diff
@@ -217,22 +241,41 @@ contract CheckSecurityConfigs is Script {
         return abi.decode(addrBytes, (address));
     }
 
-    function getAddresses(string memory addressesJsonPath) internal view returns (ProtocolAddresses memory) {
+    function getAddresses(string memory addressesJsonPath) internal view returns (ProtocolAddresses memory, bool) {
```

Comment:
I think we should rename this function now. 
Perhaps: 
`getAddressesAndFpacStatus()` I know it's clunky, but I think clarity is more important. Open to other names too.